monitoring/overlays/ocp4-stage/prometheus/prometheus.yaml

```diff
@@ -0,0 +1,158 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: prometheus
+  name: prometheus
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  podManagementPolicy: Parallel
+  selector:
+    matchLabels:
+      app: prometheus
+  template:
+    metadata:
+      labels:
+        app: prometheus
+      name: prometheus
+    spec:
+      serviceAccountName: prometheus
+      containers:
+        - name: prometheus
+          args:
+            - --storage.tsdb.retention=7d
+            - --storage.tsdb.min-block-duration=2h
+            - --storage.tsdb.max-block-duration=2h
```

Comment:
Isn't this prometheus default ?

Reply:
yes, this is the default, I got the value here, so if the workload causes an issue, we can update it.